CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/docs/layers-view.html

```diff
@@ -7,6 +7,16 @@
     </head>
     <body>
         <h1>Layers View</h1>
+        <p>
+            The Layers View holds a collection of Layers. Each Layer can represent a static set of elements, or a dynamically 
+            calculated set of elements which match a query criteria. The query structure is based on the expressions framework.
+            <br/>
+            Each layer has a visibility checkbox which determines if that layer is toggled. All elements are on the base layer 0.
+            When one or more check boxes are selected, only the elements that are members of those layers are visibly displayed. 
+            All other elements will have their <strong>visibility</strong> attribute value toggled to 0.0 (invisible).
```

Comment:
With the updates I've made to the help pages, I've actually moved away from using the strong tag just to avoid clashes with title names (which are also bold). I've moved to using double-quotes for now so if you're able to do that as well, that would be great